cartesi-rollups/api/graphql/queries/notice.mdx

```diff
@@ -10,7 +10,7 @@ Get notice based on its index
 ```graphql
 notice(
   noticeIndex: Int!
-  inputIndex: Int!
+  inputIndexWithinEpoch: Int!
```

Comment:
This field didn't change. The only field that changed was the one in the OutputValidityProof.